common/src/main/java/io/netty/util/internal/NativeLibraryLoader.java

```diff
@@ -144,7 +148,7 @@ private static boolean isWindows() {
     }
 
     private static boolean isOSX() {
-        return OSNAME.startsWith("macosx") || OSNAME.startsWith("osx");
+        return OSNAME.startsWith("osx");
```

Comment:
I just wonder why we included macosx also before. Are you sure we not need this anymore ?